overlord/snapstate/check_snap.go

```diff
@@ -53,6 +53,8 @@ var featureSet = map[string]bool{
 	"kernel-assets": true,
 	// Support for "refresh-mode: ignore-running" in snap.yaml
 	"app-refresh-mode": true,
+	// Support for "SNAP_UID" and "SNAP_EUID" environment variable
+	"snap-uid-envvar": true,
```

Comment:
now this should probably be called snap-uid-envvars

Reply:
Done.